libnavigation-base/src/main/java/com/mapbox/navigation/base/options/RerouteOptions.kt

```diff
@@ -0,0 +1,22 @@
+package com.mapbox.navigation.base.options
+
+import androidx.annotation.IntRange
+
+class RerouteOptions private constructor(
+    val avoidManeuverTime: Int
+) {
+    class Builder {
+        private var avoidManeuverTime = 8
+
+        /**
+         * Avoid maneuver time. Unit is seconds.
+         */
+        fun avoidManeuverTime(
+            @IntRange(from = 0, to = 16) avoidManeuverTime: Int
```

Comment:
Why is 16 a limit here?

Reply:
discussed: remove limits and add to docs that value mapped to distance (depends on speed) and if the distance is more than 1.000 meters it is restricted to 1.000 meters